docs/README.md

```diff
@@ -1,58 +1,81 @@
-# VTEX React App Template
+# Store Link
 
-Our guide repository to structure for react apps, that should be used as a template.
+This repository provides some blocks that can help you creating links in a store.
 
-We use `yarn` as our default package manager, before coding make sure to run yarn on: `root` and `react` folders.
+![image](https://user-images.githubusercontent.com/8517023/73387868-f1b36f80-42af-11ea-8e24-3045d2c819b4.png)
 
-## Some features:
+## Configuration
 
-### Tests
+1. Add `store-link` app to your theme's dependencies in the `manifest.json`, for example:
 
-For testing we use `@vtex/test-tools`, our own testing framework based on `react-testing-library`, the tests should be located on the `react/__tests__` folder. For references, visit our [repository](https://github.com/vtex/test-tools).
+```jsonc
+{
+  "vtex.store-link": "0.x"
+}
+```
 
-### Hooks
+Now, you are able to use all blocks exported by the `store-link` app. Checkout the full list below:
 
-Husky hooks tha runs on every `pre-commit` and `pre-push`.
+| Block name     | Description                                           |
+| -------------- | ----------------------------------------------------- |
+| `link.product` | A link a component that consumes the product context. |
+| `link`         | A normal link that doesn't consume any context.       |
 
-### Intl Equalizer
+2. Based on the exported list, choose the desired block and declare it in the block that will host the link. Below you can find an example of a link.product block being used in the [product-summary](https://vtex.io/docs/components/product/vtex.product-summary), but you can see a more complex example in the [Quick View](https://github.com/vtex-apps/modal-layout/blob/master/docs/README.md#modal-layout) that uses the `modal-layout` blocks:
```

Comment:
2. Based on the exported list, choose the desired block and declare it in the block that will host the link. Find below an example of a `link.product` being used in the [product-summary](https://vtex.io/docs/components/product/vtex.product-summary block):